src/index.html

```diff
@@ -6,8 +6,285 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
   </head>
+
   <body>
+    <!-- Header Section-->
+    <div class="header">
+      <a href="./index.html">Home</a>
+      <a href="">DOWNLOAD MY FULL RESUME</a>
+    </div>
+
+    <!-- Head Section-->
+    <div class="head">
+      <h1>John Roberts</h1>
+      <h2>BUSINESS EXPERT</h2>
+      <img src="" alt="" />
+    </div>
+
+    <!-- Slider show-->
+    <div class="slider-section">
+      <!-- AboutMe Slider-->
+      <div class="slider">
+        <div class="top">
+          <div class="title">
+            <h2>ABOUT ME</h2>
+          </div>
+          <div class="media_unique">
+            <a href=""><i class="fab fa-linkedin-in"></i></a>
+            <a href=""><i class="fab fa-twitter"></i></a>
+            <a href=""><i class="fab fa-facebook-f"></i></a>
+          </div>
+        </div>
+        <div class="bottom">
+          <div class="bottom-content">
+            <div class="intro">
+              I'm a paragraph. Click here to add your own text and edit me. It’s
+              easy. Just click “Edit Text” or double click me to add your own
+              content and make changes to the font. Feel free to drag and drop
+              me anywhere you like on your page. I’m a great place for you to
+              tell a story and let your users know a little more about you.
+            </div>
+            <div class="intro">
+              I'm a paragraph. Click here to add your own text and edit me. It’s
+              easy. Just click “Edit Text” or double click me to add your own
+              content and make changes to the font. Feel free to drag and drop
+              me anywhere you like on your page. I’m a great place for you to
+              tell a story and let your users know a little more about you.
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Experience Slider-->
+      <div class="slider">
+        <div class="top">
+          <div class="title">
+            <h2>EXPERIENCE</h2>
+          </div>
```

Comment:
`h2` is already a block element so that there is no need to be wrapped with another div tag